src/Startup/Backend/DesktopFileEnumerator.vala

```diff
@@ -66,7 +55,4 @@ class Startup.Backend.DesktopFileEnumerator : GLib.Object {
         return result;
     }
 
-    string create_path_from_name (string name) {
```

Comment:
I've removed this helper because `Utils.vala` also uses `Path.build_filename` directly.